scripts/base-deploy

```diff
@@ -3,14 +3,15 @@
 # shellcheck source=lib/common.sh
 . "${LIB_DIR:-$(cd "${0%/*}/../lib"&&pwd)}/common.sh"
 
+if [[ -z "$FAUCET" ]]; then
```

Comment:
Not sure we really need this check - we might have a network where we don't want a faucet (e.g. are we using the faucet in Kovan?)

Reply:
We are using the faucet for Kovan now (I believe).  We need this check because right now the base-deploy sets up the address export and also assigns MKR tokens to the faucet.  We would need to do a broader refactor on the base-deploy if we were going to deploy without a faucet.  I feel like that is beyond the scope of this PR and is more in line with a PR preparing the "for real" MCD deployment script